### PR TITLE
Equalizer 2.0 event API and related cleanups

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

File: livre/core/CMakeLists.txt [hunk 1]
```diff
@@ -16,7 +17,6 @@ set(LIVRECORE_PUBLIC_HEADERS
   render/GLSLShaders.h
   util/Utilities.h
   lunchboxTypes.h
-  mathTypes.h
```
Comment (bilgili) [inline livre/core/CMakeLists.txt]: I want to keep this separated. Maybe types.h can be removed from mathTypes.h and that will be more useful.

File: livre/core/CMakeLists.txt [hunk 2]
```diff
@@ -27,10 +27,6 @@ set(LIVRECORE_HEADERS
   configuration/Configuration.h
   configuration/Parameters.h
   data/DataSource.h
-  events/EventHandler.h
```
Comment (bilgili) [inline livre/core/CMakeLists.txt]: I am not sure why those are removed. I want to keep equalizer functionality separated from the code. If you have a better suggestion we can surely apply, but I would like to keep them.

File: livre/eq/EventHandler.cpp [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+#include <livre/eq/settings/RenderSettings.h>
+#include <livre/core/data/Histogram.h>
+
+namespace livre
+{
+namespace
+{
+static const int32_t SPINX_DEFAULT = 5;
+static const int32_t SPINY_DEFAULT = 5;
+static const int32_t ADVANCE_DEFAULT = 0;
```
Comment (bilgili) [inline livre/eq/EventHandler.cpp]: in anonymous namespace under cpp, you don't need the static keyword.

File: livre/eq/EventHandler.cpp [hunk 4]
```diff
@@ -0,0 +1,470 @@
+/* Copyright (c) 2016, Blue Brain Project / EPFL
```
Comment (bilgili) [inline livre/eq/EventHandler.cpp]: This file gathers so many different logics. Must be seperated imho.

File: livre/eq/EventHandler.h [hunk 5]
```diff
@@ -20,27 +21,31 @@
...
+    bool handleEvent( eq::EventType type, const eq::KeyEvent& ) override;
+    bool handleEvent( eq::EventType type, const eq::PointerEvent& ) override;
 
-    /**
-     * Executes the event function.
-     * @param eventInfo The needed information for the event. \see livre::EventInfo
-     */
-    virtual bool operator()( EventInfo& eventInfo ) = 0;
+    /** @return the frame range that the data source provides. */
+    const Vector2ui& getMaxFrameRange() const;
```
Comment (bilgili) [inline livre/eq/EventHandler.h]: This function does not feel like it belongs here :)

File: livre/core/CMakeLists.txt (hunk 2 above)
Comment (eile) [inline livre/core/CMakeLists.txt]: This PR is a suggestion:  livre/eq/EventHandler.cpp implements the Equalizer-specific event handling.

File: livre/eq/EventHandler.cpp (hunk 3 above)
Comment (eile) [inline livre/eq/EventHandler.cpp]: Moved from existing code. Will remove static.

File: livre/eq/EventHandler.cpp (hunk 4 above)
Comment (eile) [inline livre/eq/EventHandler.cpp]: It does event handling (and nothing else) and has less than 500 loc?

File: livre/eq/EventHandler.h (hunk 5 above)
Comment (eile) [inline livre/eq/EventHandler.h]: done.

**Pull request merged**